.github/aiida/profile.yaml

```
@@ -18,3 +18,4 @@ broker_password: guest
 broker_host: 127.0.0.1
 broker_port: 5672
 broker_virtual_host: ''
+test_profile: true
```

Just adding an empty new line - not necessary, but good practice.
```suggestion
test_profile: true

```

---

Yes, this will be fixed by `end-of-file-fixer`, it's been so long, no idea why I put those in excluded, seems not necessary now. 😉